packages/material-ui-lab/src/Autocomplete/Autocomplete.test.js

```diff
@@ -215,11 +215,12 @@ describe('<Autocomplete />', () => {
       );
 
       expect(container.textContent).to.equal('onetwo+1');
-      expect(getAllByRole('button')).to.have.lengthOf(4);
+      // include hidden clear button because JSDOM thinks it's visible
```

Comment:
I'm fairly certain this is due to the missing cascade support in JSDOM. I think I have a version of JSDOM implementing cascade but that's for another time. Just writing things out to improve chances that I remember it.